wallet/udb/upgrades.go

```diff
@@ -702,6 +708,49 @@ func hasExpiryFixedUpgrade(tx walletdb.ReadWriteTx, publicPassphrase []byte) err
 	return unifiedDBMetadata{}.putVersion(metadataBucket, newVersion)
 }
 
+func removeMultisigOutsWithoutTransactions(tx walletdb.ReadWriteTx, publicPassphrase []byte) error {
+	const oldVersion = 9
+	const newVersion = 10
+	metadataBucket := tx.ReadWriteBucket(unifiedDBMetadata{}.rootBucketKey())
+	txmgrBucket := tx.ReadWriteBucket(wtxmgrBucketKey)
+
+	// Assert this function is only called on version 8 databases.
+	dbVersion, err := unifiedDBMetadata{}.getVersion(metadataBucket)
+	if err != nil {
+		return err
+	}
+	if dbVersion != oldVersion {
+		const str = "removeMultisigOutsWithoutTransactions inappropriately called"
+		return apperrors.E{ErrorCode: apperrors.ErrUpgrade, Description: str, Err: nil}
+	}
+
+	var multisigOutsMarkedForDelete [][]byte
+	multisigOutBucket := txmgrBucket.NestedReadWriteBucket([]byte("ms"))
+	multisigOutBucket.ForEach(func(k, v []byte) error {
```

Comment:
I'd prefer a cursor to be used here for consistency, instead of a ForEach.  See the other upgrade functions, which all use cursors.